walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -11,13 +11,72 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
```

Comment:
This is not recommended. Only import the libraries that you will actually use:
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports